app/src/org/commcare/dalvik/application/CommCareApplication.java

```diff
@@ -776,7 +782,7 @@ private boolean getPendingUpdateStatus() {
         boolean autoUpdateTriggered = preferences.getBoolean(CommCarePreferences.AUTO_TRIGGER_UPDATE, false);
 
         if (autoUpdateTriggered) {
-            preferences.edit().putBoolean(CommCarePreferences.AUTO_TRIGGER_UPDATE, false);
+            preferences.edit().putBoolean(CommCarePreferences.AUTO_TRIGGER_UPDATE, false).commit();

```

Comment:
Missing commit means this update was never saved.
Is adding this commit going to break something?